venafi/resource_venafi_certificate.go

```diff
@@ -531,6 +562,33 @@ func enrollVenafiCertificate(ctx context.Context, d *schema.ResourceData, cl end
 	if origin == csrService {
 		req.CsrOrigin = certificate.ServiceGeneratedCSR
 	}
+
+	//setting DN values Org, Organization Units, Country, State, Locality(City)
+	country := d.Get("country").(string)
+	if country != "" {
+		req.Subject.Country = []string{country}
+	}
+	state := d.Get("state").(string)
+	if state != "" {
+		req.Subject.Province = []string{state}
+	}
+	locality := d.Get("locality").(string)
+	if locality != "" {
+		req.Subject.Locality = []string{locality}
+	}
+	org := d.Get("organization").(string)
+	if org != "" {
+		req.Subject.Organization = []string{org}
+	}
+	orUnitsNum := d.Get("organizational_units.#").(int)
+	if orUnitsNum > 0 {
```

Comment:
No need to do this check, if orUnitsNum is 0, then the for loop is ignored

Reply:
done